packages/typespec-ts/test/integration/generated/sharedRoute/src/clientDefinitions.ts

```diff
@@ -24,6 +24,7 @@ import {
 import { Client, StreamableMethod } from "@azure-rest/core-client";
 
 export interface ListByResourceGroup {
+  /** different query & header parameters */
```

Comment:
Introduced by new feature: 
https://microsoft.github.io/typespec/release-notes/release-2023-06-06#support-for-doc-comments